dev/run_examples.sh

```diff
@@ -1,18 +1,15 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 # This bash file is to run all .py examples files in a folder.
-# It should be run from the root folder example where there is a .toml file
-# This script first will run `poetry lock` and `poetry install` to update example folder installation
-# Then will search for `*.py` files inside root folder and subdirectories and it will run them with `poetry run python <file.py>`
+# It should be run from an example folder containing a `pyproject.toml`.
 
-poetry lock
-poetry install
```

Comment:
I removed the poetry bits because I wanted to use this script in the pipeline for the looping part below. Let me know if you would prefer that we keep it somehow.

Reply:
It's okey